Artifacts/windows-test-paramtypes/artifactfile.json

```diff
@@ -1,6 +1,6 @@
 {
   "$schema": "https://raw.githubusercontent.com/Azure/azure-devtestlab/master/schemas/2016-11-28/dtlArtifacts.json",
-  "title": "Test Parameter Types",
+  "title": "[leov] Test Parameter Types",
```

Comment:
We don't want your alias in the title, do we?

Reply:
Oops! 😉